sdk/core/core/src/az_span.c

```diff
@@ -226,74 +211,55 @@ AZ_INLINE void _az_uint8_swap(uint8_t* a, uint8_t* b)
 }
 
 /**
- * @brief move the content from span @b a to span @b b and viceverse.
+ * @brief move the content from span @span1 to span @span2 and viceverse.
  * The smallest span (less content) is swapped only. Example
- * a = 111
- * b = 22
+ * span1 = 111
+ * span2 = 22
  * after swaping
- * a = 221
- * b = 11
+ * span1 = 221
+ * span2 = 11
  *
  * @param a source/destination span
  * @param b destination/source span
  */
-void _az_span_swap(az_span a, az_span b)
+void _az_span_swap(az_span span1, az_span span2)
```

Comment:
Do we need this method? Doesn't look like we have any callers.

Reply:
Great - if we have no caller then we should get rid of it. I know we have dead code in the product. We're hoping that code coverage will make this obvious and then we can get rid of stuff. Good catch

Reply:
removed